src_py/sysfont.py

```diff
@@ -58,26 +58,37 @@ def initsysfonts_win32():
     fonts = {}
 
     # add fonts entered in the registry
-    key_path = "SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Fonts"
-    key = _winreg.OpenKey(_winreg.HKEY_LOCAL_MACHINE, key_path)
+    MICROSOFT_FONT_DIRS = [
+        "SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion\\Fonts",
+        "SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Fonts",
+    ]
 
-    for i in range(_winreg.QueryInfoKey(key)[1]):
+    for domain in [
+        _winreg.HKEY_LOCAL_MACHINE,
+        _winreg.HKEY_CURRENT_USER,
+    ]:
         try:
-            # name is the font's name e.g. Times New Roman (TrueType)
-            # font is the font's filename e.g. times.ttf
-            name, font, _ = _winreg.EnumValue(key, i)
-        except OSError:
-            break
-
-        if splitext(font)[1].lower() not in OpenType_extensions:
+            key = _winreg.OpenKey(domain, MICROSOFT_FONT_DIRS[0])
+        except FileNotFoundError:
             continue
-        if not dirname(font):
-            font = join(fontdir, font)
 
-        # Some are named A & B, both names should be processed separately
-        # Ex: the main Cambria file is marked as "Cambria & Cambria Math"
-        for name in name.split("&"):
-            _parse_font_entry_win(name, font, fonts)
+        for i in range(_winreg.QueryInfoKey(key)[1]):
+            try:
+                # name is the font's name e.g. Times New Roman (TrueType)
+                # font is the font's filename e.g. times.ttf
+                name, font, _ = _winreg.EnumValue(key, i)
+            except OSError:
+                break
+
+            if splitext(font)[1].lower() not in OpenType_extensions:
+                continue
+            if not dirname(font):
+                font = join(fontdir, font)
+
+            # Some are named A & B, both names should be processed separately
+            # Ex: the main Cambria file is marked as "Cambria & Cambria Math"
+            for name in name.split("&"):
```

Comment:
I am pretty sure this could cause problem. Why? This can happen when you have a .ttc file. In brief, it is a file that contains a collection of font. Because if that, it can have multiple name and there are NOT ALWAYS separate with &.

Reply:
Resolving this. We didn't have special case code for ttc before and we don't now, but that's beyond the scope of this PR.